### PR TITLE
closes #494 - Fix setStar resolver so people can't give themselves a star

### LINKED ISSUE #494: [bug] people can give themself a star, the resolver does not have checks for self-giving a star
The setStar resolver does not check if the mentor and student are the same. People shouldn't be able to give themselves stars

### PR DESCRIPTION
**Pull request by anthonykhoa**
closes #494 

Right now, people can give themselves a star, because the `setStar` resolver does not check for that edge case. 

## This PR will:
* Throw and error if someone tries to give themself a star, by checking if `studentId === mentorId`
* Update test file to reflect change in setStar resolver

### CODE REVIEW

File: helpers/controllers/starsController.ts [hunk 1]
```diff
@@ -17,6 +17,10 @@ export const setStar = async (
   try {
     const studentId = validateStudentId(req)
     const { lessonId, mentorId } = arg
+
+    if (studentId === mentorId) {
+      throw new Error('Unable to give error to yourself')
```
Comment (ggwadera) [inline helpers/controllers/starsController.ts:22]: Unable to give **ERROR** to yourself

🤔

Shouldn't it be star?

Comment (anthonykhoa) [reply helpers/controllers/starsController.ts:22]: Ohoho good catch :)

**Review by ggwadera — APPROVED** [review]

**Review by hwong0305 — APPROVED** [review]
